packages/clay-color-picker/src/index.tsx

```diff
@@ -174,9 +193,15 @@ const ClayColorPicker: React.FunctionComponent<IProps> = ({
 							useNative ? onValueChange(e.target.value) : null
 						}
 						ref={valueInputRef}
-						style={{display: 'none'}}
+						style={{
+							height: 0,
+							margin: 0,
+							padding: 0,
+							visibility: 'hidden',
+							width: 0,
+						}}
 						type={useNative ? 'color' : 'text'}
-						value={value}
+						value={`#${hexInputValue}`}
```

Comment:
@bryceosterhaus wondering if with the addition of the hash we could break some behaviors that expected the hex value to come without the hash. I see this in DDM Field with ColorPicker but I'm not sure if it would be a problem for the backend to handle it.

Reply:
Apparently there seems to be no problem, I don't see any special validation for that.

Reply:
Yeah this is could technically break some uses, but we were using the wrong html spec anyways so its really a fix for us. I was going to go through liferay-portal and update all instances when I release

Reply:
Makes sense, no problem. LGTM.